account/src/main/java/run/chronicle/account/dto/CreateAccount.java

```diff
@@ -19,107 +19,139 @@
 package run.chronicle.account.dto;
 
 import net.openhft.chronicle.core.io.InvalidMarshallableException;
-import net.openhft.chronicle.wire.converter.Base85;
+import net.openhft.chronicle.wire.converter.ShortText;
 
 /**
- * Represents a specific kind of event: the creation of an account.
- * This class extends the abstract event class and adds properties specific to the creation of an account:
- * name, account, currency, and balance. The setters follow a fluent style.
+ * Represents the event of creating a new account.
+ * This class extends {@link AbstractEvent} and adds properties specific to account creation,
+ * such as the account holder's name, account number, currency, balance, and overdraft limit.
+ * Setters are designed using the fluent interface pattern for method chaining.
  */
 public class CreateAccount extends AbstractEvent<CreateAccount> {
     private String name; // Name associated with the account
     private long account; // Account identifier
-    @Base85
-    private int currency; // Currency for the account, represented in Base85
-    private double balance; // Balance of the account
+    @ShortText
+    private int currency; // Currency for the account
+    private double balance; // Initial Balance of the account
+    private double overdraft; // Overdraft limit of the account
 
     /**
+     * Retrieves the account identifier.
+     *
      * @return the account identifier
      */
     public long account() {
         return account;
     }
 
     /**
-     * Sets the account identifier and returns the updated object.
+     * Sets the account identifier.
      *
      * @param account the account identifier to set
-     * @return the updated object
+     * @return this object for method chaining
      */
     public CreateAccount account(long account) {
         this.account = account;
         return this;
     }
 
     /**
-     * @return the name of the account
+     * Retrieves the name associated with the account.
+     *
+     * @return the account holder's name
      */
     public String name() {
         return name;
     }
 
     /**
-     * Sets the name and returns the updated object.
+     * Sets the name associated with the account.
      *
      * @param name the name to set
-     * @return the updated object
+     * @return this object for method chaining
```

Comment:
minor, but in 99% of our code when we return this for a fluent API we just have
```
* @return this
```
as our javadoc

Reply:
Probably sufficient but minimal.